src/pages/blog/work-study:-how-to-get-work-experience.md

```diff
@@ -0,0 +1,69 @@
+  Check out these tips we curated for you to use.
+tags:
+  - Interview preparation
+  - Job interview
+featured: true
+image: /img/work-learning.jpg
+category: Career
+parentcategory: Career
+public: true
+---
+
+Students and recent grads, we hear you: transitioning into the working world is hard, especially for those who prioritized grades over gaining work experience. Many people are caught in the catch 22 of needing experience to get a job, and needing a job to get experience. There are ways to get around this, and we’re here to help you out! So keep reading for our tips on how to gain experience that will make you stand out to employers. All you need is a little creativity, work ethic and entrepreneurial spirit! 
+
+
+ **1.)  Join/ start a club related to the industry that you want to go into:**
+
+Look for school clubs and organizations related to your field. Aim for a leadership position and find ways to demonstrate your abilities in a measurable way. It could be increasing club signups, organizing events, generating sponsorship, or handling social media. Think of how you can make a positive impact and learn at the same time, and don’t forget to hype it up on your resume. 
+
+Club doesn’t exist? This is the perfect chance to show your entrepreneurial skills by starting your own club, and you can conveniently place yourself in an executive position. Invite leaders to come speak at your club events and create your own networking opportunities. 
+
+
+**2.)  Write and publish independent work about your field:**
+
+Another way to create your own work experience is to generate your own content. Consider starting a blog about your industry, or writing articles and submitting them to publications. This will show recruiters that you took the time and effort to learn about your domain. It also promotes your research and writing abilities. 
+When marketed correctly, the content that you create is a valuable addition to your resume or portfolio. 
+
+
+**3.) Build an online portfolio:**
+
+On a related note, you can compile school projects, independent works, articles that you’ve written and more into a portfolio. Keep it relevant and streamlined to the job you want, and you will impress recruiters. Building an online portfolio also gives you the chance to brand yourself and can lead the way to networking opportunities. Always make sure to put the link near the top of your resume! 
+
+
+**4.) Build a product or service that people want to use:**
+
+Starting your own business or startup is a great way to accelerate your learning and build work skills. This can be a challenge while still at school, but it can be extremely rewarding when you’re trying to sell yourself in the job market.  You probably didn’t know that Canadian schools are incentivised to help students become entrepreneurs, so find out what your school has to offer! 
+Bonus: you might become so successful that finding a job might not be a concern any more.
+
+
+**5.) Join a startup:**
```

Comment:
@charlesjavelona Can we include some media in these posts (pictures, gifs, etc) to break up the monotony?

Reply:
We could, I just don't know how and we could do it for the other ones as well. I just need something up now that Trevor and Moe can use. I will add more later.

Reply:
oh wait, I know how

Reply:
It's annoying to do.

Reply:
Gotcha. It would really improve the readability of these blog posts and make them a bit more attractive to read; lower bounce rates. 

You can do this the same way you include images in markdown on GitHub or Gitlab. https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet